pkg/v1/tkg/client/machine_deployment.go

```diff
@@ -617,10 +619,18 @@ func populateVSphereMachineTemplate(machineTemplate *vsphere.VSphereMachineTempl
 			Devices: []vsphere.NetworkDeviceSpec{
 				{
 					NetworkName: options.VSphere.Network,
-					DHCP4:       true,
 				},
 			},
 		}
+		if len(options.VSphere.Nameservers) > 0 {
+			machineTemplate.Spec.Template.Spec.Network.Devices[0].Nameservers = options.VSphere.Nameservers
```

Comment:
Why are modifying only Devices[0] here?

Reply:
Our default configuration only has one network device. This code specifies exactly one new device to be set on the VSphereMachineTemplate. I think it would take a significant feature request to expand the number of devices in our preconfigured plans and I'm really not sure how we'd correctly handle setting up the networking in those situations.

Reply:
Cool. Thanks!